common/src/main/java/com/wynntils/models/players/scoreboard/PartyFinderScoreboardPart.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.players.scoreboard;
+
+import com.wynntils.handlers.scoreboard.ScoreboardPart;
+import com.wynntils.handlers.scoreboard.ScoreboardSegment;
+import com.wynntils.handlers.scoreboard.type.SegmentMatcher;
+
+public class PartyFinderScoreboardPart extends ScoreboardPart {
+    private static final SegmentMatcher PARTY_FINDER_MATCHER = SegmentMatcher.fromPattern("Party Finder:");
```

Comment:
Is there a Party Finder scoreboard part? I've never seen that? How, when does it appear?

Reply:
Whenever you queue for a party finder queue. 
![image](https://user-images.githubusercontent.com/49001742/229340541-3e8cd2ed-91fe-4009-84ad-c166b56a8d4f.png)

Reply:
Oh, I guess I have never been queuing, only joining existing parties. At least not enough to think about that.